flake.nix

```diff
@@ -26,7 +26,7 @@
     # over time.  NOTE: This file is not permanent, find the available one at
     # https://download.geofabrik.de/asia/india/
     # NOTE: If you change this, also change `openStreetDataFileName` in osrm.nix
-    osrm-pbf.url = "https://download.geofabrik.de/asia/india/southern-zone-230101.osm.pbf";
+    osrm-pbf.url = "https://download.geofabrik.de/asia/india/southern-zone-231218.osm.pbf";
```

Comment:
Why was this reverted?

cc @vsaimanohar 

https://github.com/nammayatri/nammayatri/pull/4783